### DISCUSSION TITLE
I don't know what's going on

### DISCUSSION THREAD

**Discussion by NeGoblin** [danielkrupinski/Osiris#3822] [Q&A]
I've been using Osiris for a long time and I'm very grateful to the developers. But recently noticed that there is a bug with walking on the record and from the side I walk very twitchy or like on rails. If anyone knows how to fix it write, would be very grateful.

**Comment by diekunstderfuge** [comment]
I don't know if it is the same thing you are talking about, but I noticed that when I watch the replays of my deaths and I am using some character skin. My character moves very strange and I don't know if other people see that too.
But recently I have seen other people moving that way too.

**Reply by NeGoblin** [reply]
Yes, that's exactly it. It's not about skin I have this even when nothing is on
